exams/tasks.py

```diff
@@ -55,3 +55,37 @@ def export_exam_profiles():
             (ExamProfile.objects
              .filter(id__in=exam_profile_ids)
              .update(status=ExamProfile.PROFILE_INVALID))
+
+
+@async.task
+def export_exam_authorizations():
+    """
+    Sync any outstanding profiles
+    """
+    exam_authorizations = (ExamAuthorization.objects
+                           .filter(status=ExamAuthorization.STATUS_PENDING)
+                           .prefetch_related('user__profile', 'course__program'))
```

Comment:
nit: this can be written like
```
exam_authorizations = ExamAuthorization.objects.filter(
    status=ExamAuthorization.STATUS_PENDING).prefetch_related('user__profile', 'course__program')
```